config/config.txt

```diff
@@ -530,9 +530,9 @@ DISABLE_HIGH_POP_MC_MODE_AMOUNT 100
 ## Uncomment to set the number of /world/Reboot()s before the DreamDaemon restarts itself. 0 means restart every round. Requires tgstation server tools.
 ROUNDS_UNTIL_HARD_RESTART 3
 
-##Default screen resolution, in tiles.
+##Default view size, in tiles.
 ##	By default, this is 15x15, which gets simplified to 7 by BYOND, as it is a 1:1 screen ratio.
-##	For reference, Goonstation uses a resolution of 21x15 for it's widescreen mode.
+##  15x15 would be the standard square view. 21x15 is what goonstation uses for widescreen.
```

Comment:
This also seems like an irrelevant change

Reply:
Leftover Accident